standardization/src/test/scala/za/co/absa/enceladus/standardization/fixtures/CsvSpecialCharsFixture.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.standardization.fixtures
+
+import java.io.{DataOutputStream, File, FileOutputStream}
+import java.nio.charset.StandardCharsets
+
+import org.apache.spark.sql.types._
+import org.scalatest.{BeforeAndAfterAll, Suite}
+import za.co.absa.enceladus.model.Dataset
+
+trait CsvSpecialCharsFixture extends BeforeAndAfterAll {
+
+  this: Suite =>
+
+  val csvContent: String =
```

Comment:
I would put these (`csvContent`, `schema`, `inputFileName`, `dataSet`) with the test they belong to.
1. It's difficult to see what is being tested when I need to switch files.
2. It's not really the `Fixture`'s place to hold this data.

Reply:
Do you mean to put everything from this file to the suite itself?
The fixture can then be 'with temp file' accepting the files content and charset, right?